src/components/search/Search.tsx

```diff
@@ -0,0 +1,40 @@
+// tslint:disable-next-line:no-submodule-imports
+import 'bootstrap/dist/css/bootstrap.min.css';
+import React, { ChangeEvent, Component, FormEvent } from 'react';
+import { INote } from '../../interfaces/note';
+
+type MyProps = { handleSearch: (searchValue: string) => void };
+type MyState = { searchValue: string };
+
+export default class Note extends Component<MyProps, MyState> {
+    constructor(props: MyProps) {
+        super(props);
+        this.state = {
+            searchValue: '',
+        };
+
+        this.handleSearchChange = this.handleSearchChange.bind(this);
+    }
+
+    render() {
+    return (
+            <div className="form-group">
+                <input
+                    onChange={this.handleSearchChange}
+                    name="search"
+                    value={this.state.searchValue}
+                    className="form-control form-control-lg"
+                    type="text"
+                    placeholder="Search"
+                />
+            </div>
+    );
+  }
+
+    private handleSearchChange(event: ChangeEvent<HTMLInputElement>) {
+        this.setState({
+        searchValue: event.target.value,
+        });
```

Comment:
Indentation.

Also there's a bug with the search if you clear the input using select+delete:

![clear_input_bug](https://user-images.githubusercontent.com/16776066/52785634-24456780-3061-11e9-80c2-d67b5c3b5183.gif)

Reply:
fixed :)